embeddings/pipeline/lightning_classification.py

```diff
@@ -39,8 +41,19 @@ def __init__(
         model_config_kwargs: Optional[Dict[str, Any]] = None,
         predict_subset: LightingDataModuleSubset = LightingDataModuleSubset.TEST,
     ):
+        output_path = Path(output_path)
+        task_train_kwargs = (
+            task_train_kwargs if task_train_kwargs else self.DEFAULT_TASK_TRAIN_KWARGS
+        )
+        task_model_kwargs = (
+            task_model_kwargs if task_model_kwargs else self.DEFAULT_TASK_MODEL_KWARGS
+        )
+        tokenizer_name = tokenizer_name if tokenizer_name else embedding_name
+        datamodule_kwargs = datamodule_kwargs if datamodule_kwargs else {}
+        model_config_kwargs = model_config_kwargs if model_config_kwargs else {}
```

Comment:
note that this is changed in #173 as these kwargs get updated now

Reply:
I have commented on changes in #173 and if I am right, this is a proper solution.